### DISCUSSION TITLE
BTRM and YERB

### DISCUSSION THREAD

**Discussion by magecrimson** [coinfoundry/miningcore#1326] [Q&A]
Are you aware of any issues having both BTRM and YERB running in the same pool.  It seems each time I add YERB to the pool, coins are mined perfectly fine but payments break.  YERB states not enough funds in wallet which is false and BTRM payment just break, and the % of confirmations for new found blocks stops in its tracks.

See example below: YERB error

[yerb] [Bitcoin Payout Handler] sendmany returned error: Account has insufficient funds code -6

It is almost as if the payment handler can not figure out the difference between btrm and yerb.  

Any direction would be great!!  Thanks

**Comment by oliverw** [comment]
I doubt anyone can help you with this as neither coin is part of the official releases.

**Reply by magecrimson** [reply]
Oliver, yerb is in the release as part of the coins.json in the existing repo.  Both yerb and Btrm are forks of RTM which is also in the repo.  The only coin not in the existing repo is btrm, though should work out of the box as yerb is a fork of rtm as well as btrm.  I am kinda at a loss at this point.

**Reply by magecrimson** [reply]
This can be marked as resolved.  Solution was found deep in other discussions on the web: Solution below:

The pool uses the sendmany RPC command with an empty "" string for the "fromaccount"
If you labeled any of the addresses you put in the pools.json it will fail as it can not read it and shows the "" address as a -
to resolve you need to use the setaccount command to null the label
like this:   setaccount <address> ""
so if you come across it or someone else does feel free to pass the info

**Reply by oliverw** [reply]
> Oliver, yerb is in the release as part of the coins.json in the existing repo. Both yerb and Btrm are forks of RTM which is also in the repo. The only coin not in the existing repo is btrm, though should work out of the box as yerb is a fork of rtm as well as btrm. I am kinda at a loss at this point.

Oh my bad. Glad that you got it resolved.
